src/components/util/equipment-labeling-toggle.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import React from 'react';
+
+import { makeStyles, withStyles } from '@material-ui/core/styles';
+
+import ListItemText from '@material-ui/core/ListItemText';
+import ToggleButton from '@material-ui/lab/ToggleButton';
+import ToggleButtonGroup from '@material-ui/lab/ToggleButtonGroup';
+import MenuItem from '@material-ui/core/MenuItem';
+import Typography from '@material-ui/core/Typography';
+import { FormattedMessage } from 'react-intl';
+
+const useStyles = makeStyles(() => ({
+    sizeLabel: {
+        fontSize: '16px',
+    },
+    toggleDisplay: {
+        marginLeft: '15px',
+        pointerEvents: 'auto',
+    },
+    toggleButton: {
+        height: '30px',
```

Comment:
[Jon Harper mode ON] Do you really need to hard code the toggle high and all other values in px? [Jon Harper mode OFF]